net: xfrm: export 'xfrm_outer_mode_output'

Export 'xfrm_outer_mode_output' symbol for use in XFRM output fastpath code.

--- a/include/net/xfrm.h
+++ b/include/net/xfrm.h
@@ -1684,6 +1684,8 @@ int xfrm_trans_queue(struct sk_buff *skb
 int xfrm_output_resume(struct sock *sk, struct sk_buff *skb, int err);
 int xfrm_output(struct sock *sk, struct sk_buff *skb);
 
+int xfrm_outer_mode_output(struct xfrm_state *x, struct sk_buff *skb);
+
 #if IS_ENABLED(CONFIG_NET_PKTGEN)
 int pktgen_xfrm_outer_mode_output(struct xfrm_state *x, struct sk_buff *skb);
 #endif
--- a/net/xfrm/xfrm_output.c
+++ b/net/xfrm/xfrm_output.c
@@ -450,7 +450,7 @@ static int xfrm6_prepare_output(struct x
 	return -EAFNOSUPPORT;
 }
 
-static int xfrm_outer_mode_output(struct xfrm_state *x, struct sk_buff *skb)
+int xfrm_outer_mode_output(struct xfrm_state *x, struct sk_buff *skb)
 {
 	switch (x->props.mode) {
 	case XFRM_MODE_BEET:
@@ -478,6 +478,7 @@ static int xfrm_outer_mode_output(struct
 
 	return -EOPNOTSUPP;
 }
+EXPORT_SYMBOL_GPL(xfrm_outer_mode_output);
 
 #if IS_ENABLED(CONFIG_NET_PKTGEN)
 int pktgen_xfrm_outer_mode_output(struct xfrm_state *x, struct sk_buff *skb)
